quick-start/src/main/ui/app/jobs/job-export.component.ts

```diff
@@ -51,8 +43,4 @@ export class JobExportDialogComponent {
         });
```

Comment:
@alexsmr Here's the line where the generic call to the dialogService.alert() is called.